api/src/main/java/org/openmrs/module/fhir2/api/util/LastnResult.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.fhir2.api.util;
+
+/**
+ * The class stores the result of $lastn type operations
+ */
+import java.util.Date;
+import java.util.Map;
+
+import lombok.Data;
+
+@Data
+public class LastnResult {
+	
+	private final String uuid;
+	
+	private final Date datetime;
+	
+	private final Map<String, Object> attributes;
+	
+	/**
+	 * Constructor for observations in $lastn operation
```

Comment:
We should have a more generic description for these constructors since they might be used for other resources as well in the future.